config/jobs/kubernetes/sig-scalability/sig-scalability-golang.yaml

```diff
@@ -51,8 +51,8 @@ periodics:
       - --cluster=gce-golang
       - --env=CL2_ENABLE_PVS=false
       - --env=CL2_LOAD_TEST_THROUGHPUT=50
-      - --env=KUBEMARK_CONTROLLER_MANAGER_TEST_ARGS="--profiling --kube-api-qps=200 --kube-api-burst=200"
-      - --env=KUBEMARK_SCHEDULER_TEST_ARGS="--profiling --kube-api-qps=200 --kube-api-burst=200"
+      - --env=KUBEMARK_CONTROLLER_MANAGER_TEST_ARGS=--profiling --kube-api-qps=200 --kube-api-burst=200
+      - --env=KUBEMARK_SCHEDULER_TEST_ARGS=--profiling --kube-api-qps=200 --kube-api-burst=200
```

Comment:
Are you sure it will work without quotes?

Reply:
Won't it leave only the first item?

Reply:
Given that something like this https://github.com/kubernetes/test-infra/blob/1b7a5c47abf1554895e5aaa464c64db2f5212fa1/config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml#L35
works, I think it should work as well